### PR TITLE
Apply SmoothPointerDataDispatcher to Fuchsia

### PR DESCRIPTION
**Pull request by liyuqian**
Fixes https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=35050
and https://github.com/flutter/flutter/issues/38811

### CODE REVIEW

**Comment by liyuqian** [comment]
FYI, this is landing without a Fuchsia-specific test because of https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=35050#c49

**Comment by chinmaygarde** [comment]
The presubmit failures are because you added the function to the wrong platform_view.cc

File: shell/common/platform_view.cc [hunk 1]
```diff
@@ -137,4 +137,10 @@ void PlatformView::SetNextFrameCallback(const fml::closure& closure) {
   delegate_.OnPlatformViewSetNextFrameCallback(closure);
 }
 
+PointerDataDispatcherMaker PlatformView::GetDispatcherMaker() {
+  return [](DefaultPointerDataDispatcher::Delegate& delegate) {
+    return std::make_unique<SmoothPointerDataDispatcher>(delegate);
+  };
+}
```
Comment (gw280) [inline shell/common/platform_view.cc]: Did you mean to put this in the platform_view.cc for Fuchsia, not the common one?

**Comment by liyuqian** [comment]
@chinmaygarde @gw280 : thanks for the remind! Just fixed. Thank you and our presubmit tests so much.

**Pull request merged**